.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+version: 2.1
```

Comment:
@dantevvp Just curious - where is the high-level summary of changes we needed to make in the circle config to get this working?